### PR TITLE
FIX: Only show and let vote on bars nearby + UI fixes

### PR DESCRIPTION
**Pull request by joklek**
- Improved sorting code quality. Also solved a few crashes with them
- Improve UI: responsive input validation with regex
- Improve UI: beautiful beer meter by @DomantasDev 
- Move files to folders
- Rating and fetching now works with bars nearby

I've added a few comments to the code. If you have any solutions to the problems I see, feel free to push to this branch.

### CODE REVIEW

File: BeerOverflowWindowsApp/BarComparers/ComparerByDistance.cs [hunk 1]
```diff
@@ -21,11 +21,11 @@ public int Compare(BarData barData1, BarData barData2)
             }
             else if (distance1 < distance2)
             {
```
Comment (joklek) [inline BeerOverflowWindowsApp/BarComparers/ComparerByDistance.cs]: Inverted the values so it worked as I wanted. Looking at it now, it's a hacky solution :/

File: BeerOverflowWindowsApp/BarRating.cs [hunk 2]
```diff
@@ -8,92 +8,90 @@ namespace BeerOverflowWindowsApp
...
                     throw new ArgumentOutOfRangeException(nameof(compareType), compareType, null);
             }
         }
+
+        private void SortAndInvertIfNeeded(IComparer<BarData> comparer, CompareType compareType)
+        { 
+            if (BarsData != null)
+            {
+                BarsData.BarsList.Sort(comparer);
+                if (compareType == _lastCompare)
```
Comment (joklek) [inline BeerOverflowWindowsApp/BarRating.cs]: This logic works but is not the best one. Try multiple searches of the same coordinates, the list inverts everytime.

File: BeerOverflowWindowsApp/BarComparers/ComparerByDistance.cs (hunk 1 above)
Comment (CactusGreen) [reply BeerOverflowWindowsApp/BarComparers/ComparerByDistance.cs]: I think this is fine. The whole point of iComparer is to sort it the way we need it to.

File: BeerOverflowWindowsApp/BarRating.cs (hunk 2 above)
Comment (CactusGreen) [reply BeerOverflowWindowsApp/BarRating.cs]: Fixed inversion

**Pull request merged**